### PR TITLE
0.8 RelNotes

### PR DESCRIPTION
**Pull request by bromeg**
Updated RelNotes for 0.8, waiting on package # (marked TO ADD), will make that change tomorrow when we have it

### CODE REVIEW

File: articles/relnotes/index.md [hunk 1]
```diff
@@ -22,6 +25,24 @@ uid: microsoft.quantum.relnotes
...
+
+This release contains the following:
+
+- New indexing locations for slicing arrays, [click here for more information](xref:microsoft.quantum.language.expressions#array-slices)
+- Added Dockerfile hosted on the [Microsoft Container Registry](https://github.com/microsoft/ContainerRegistry), [click here for more information](https://github.com/microsoft/iqsharp/blob/master/README.md)
+- Breaking change for TraceSimulator, update to configuration settings, name changes, [click here for more information](https://docs.microsoft.com/en-us/dotnet/api/microsoft.quantum.simulation.simulators.qctracesimulators.qctracesimulatorconfiguration?view=quantum-dotnet-latest)
+
+See the full list of closed PRs for [libraries](https://github.com/Microsoft/QuantumLibraries/pulls?q=is%3Apr+is%3Aclosed) and [samples](https://github.com/Microsoft/Quantum/pulls?q=is%3Apr+is%3Aclosed).  
+
+The changes are summarized here as well as instructions for upgrading your existing programs.  Read more about these changes on the [Q# dev blog](https://devblogs.microsoft.com/qsharp).
```
Comment (tcNickolas) [inline articles/relnotes/index.md]: Are we planning to write a blog post about 0.8 for the dev blog?

File: articles/relnotes/index.md [hunk 2]
```diff
@@ -108,7 +129,10 @@ We added a [sample of using Q# with F# driver](https://github.com/Microsoft/Quan
 ## Migrating existing projects to 0.6.1905.
 
   
-1.  Follow [these instructions](xref:microsoft.quantum.install#updating-iq) to update IQ#.  
+1.  To update IQ# to the latest version, from the command line, execute:
+```Command Prompt
+dotnet tool update -g Microsoft.Quantum.IQSharp
```
Comment (anpaz-msft) [inline articles/relnotes/index.md]:
```suggestion
dotnet tool update -g Microsoft.Quantum.IQSharp
dotnet iqsharp install --user
```

File: articles/relnotes/index.md [hunk 3]
```diff
@@ -22,6 +25,24 @@ uid: microsoft.quantum.relnotes
...
 > * References between projects migrated to 0.6 and higher and projects using older package versions are **not** supported.
 
+# Version 0.8 (*PackageReference 0.8.1907.1701*)
+
+*Release date: July 12, 2019*
+
+This release contains the following:
+
+- New indexing locations for slicing arrays, [click here for more information](xref:microsoft.quantum.language.expressions#array-slices)
+- Added Dockerfile hosted on the [Microsoft Container Registry](https://github.com/microsoft/ContainerRegistry), [click here for more information](https://github.com/microsoft/iqsharp/blob/master/README.md)
```
Comment (cgranade) [inline articles/relnotes/index.md]: Strictly speaking, the Dockerfile isn't hosted on MCR, but defines how to build the image that we host on MCR. For SEO, I'd also suggest changing the text for the IQ# link:
```suggestion
- Added Dockerfile hosted on the [Microsoft Container Registry](https://github.com/microsoft/ContainerRegistry), see the [IQ# repository for more information](https://github.com/microsoft/iqsharp/blob/master/README.md)
```

File: articles/relnotes/index.md [hunk 4]
```diff
@@ -22,6 +25,24 @@ uid: microsoft.quantum.relnotes
...
+- Breaking change for TraceSimulator, update to configuration settings, name changes, [click here for more information](https://docs.microsoft.com/en-us/dotnet/api/microsoft.quantum.simulation.simulators.qctracesimulators.qctracesimulatorconfiguration?view=quantum-dotnet-latest)
```
Comment (cgranade) [inline articles/relnotes/index.md]:
```suggestion
- Breaking change for [the trace simulator](xref:microsoft.quantum.machines.qc-trace-simulator.intro), update to configuration settings, name changes; see the [.NET API Browser for the updated names](https://docs.microsoft.com/dotnet/api/microsoft.quantum.simulation.simulators.qctracesimulators.qctracesimulatorconfiguration).
```

**Comment by anpaz-msft** [comment]
@cgranade or @tcNickolas , can we please merge this into master? want to push it live as part of the release. thanks!

File: articles/relnotes/index.md [hunk 5]
```diff
@@ -25,6 +25,23 @@ dotnet tool update -g Microsoft.Quantum.IQSharp
 > * In order to work with a solution in Visual Studio, all projects contained in the solution need to be in the same folder as the solution or in one of its subfolders.  
 > * References between projects migrated to 0.6 and higher and projects using older package versions are **not** supported.
 
+# Version 0.8 (*PackageReference 0.8.1907.1701*)
+
+*Release date: July 12, 2019*
+
+This release contains the following:
+
+- New indexing locations for slicing arrays, [click here for more information](xref:microsoft.quantum.language.expressions#array-slices)
```
Comment (cgranade) [inline articles/relnotes/index.md]:
```suggestion
- New indexing locations for slicing arrays, [see the language reference](xref:microsoft.quantum.language.expressions#array-slices) for more information.
```

**Pull request merged**